lib/stm32/g4/Makefile

```diff
@@ -51,6 +51,7 @@ OBJS += rng_common_v1.o
 OBJS += spi_common_all.o spi_common_v2.o
 OBJS += timer_common_all.o timer_common_f0234.o
 OBJS += quadspi_common_v1.o
+OBJS += usart_common_v2.o usart_common_all.o
```

Comment:
you've added the usart objects, but not the crc objects?

Reply:
Added CRC objects